cypher.go

```diff
@@ -14,10 +14,12 @@ import (
 // unmarshalled into it when the query is executed. Result must be a pointer
 // to a slice of structs - e.g. &[]someStruct{}.
 type CypherQuery struct {
-	Statement  string                 `json:"statement"`
-	Parameters map[string]interface{} `json:"parameters"`
-	Result     interface{}            `json:"-"`
-	cr         cypherResult
+	Statement    string                 `json:"statement"`
+	Parameters   map[string]interface{} `json:"parameters"`
+	Result       interface{}            `json:"-"`
+	cr           cypherResult
+	IncludeStats bool `json:"includeStats"`
+	Stats        Stats
```

Comment:
`Stats` is always populated by data returned from Neo4j; there is no use case for populating it manually.  

Maybe therefore it would be better to have make the stats available as a method rather than a variable.  Maybe something with a signature like this:

```go
func (cq *CypherQuery) Stats() (*Stats, error) 
```

This one would return an error if the stats were not retrieved with a query.  Do you think that's overkill, or a useful feature?